sentinel-core/src/test/java/com/alibaba/csp/sentinel/TracerTest.java

```diff
@@ -8,6 +9,24 @@
  */
 public class TracerTest extends Tracer {
 
+    @Test
+    public void testTraceWhenContextSizeExceedsThreshold() {
+        int i = 0;
+        for (; i < Constants.MAX_CONTEXT_NAME_SIZE; i++) {
+            ContextUtil.enter("test-context-" + i);
+            ContextUtil.exit();
+        }
+
+        try {
+            ContextUtil.enter("test-context-" + i);
+            throw new RuntimeException("test");
+        } catch (Exception e) {
+            Tracer.trace(e);
+        } finally {
+            ContextUtil.exit();
+        }
```

Comment:
Maybe we need to do some context clean-up here (using [ContextTestUtil](https://github.com/alibaba/Sentinel/blob/master/sentinel-core/src/test/java/com/alibaba/csp/sentinel/context/ContextTestUtil.java))?

Reply:
Yes, it's needed. Very precise!